docs/developers/sdk/index.md

```diff
@@ -0,0 +1,108 @@
+---
+id: "index"
+title: "@audius/sdk"
+sidebar_label: "Readme"
+sidebar_position: 0
+custom_edit_url: null
+---
+
+# Audius JavaScript SDK
+
+## Overview
+
+The Audius JavaScript (TypeScript) SDK allows you to easily build upon and interact with the Audius protocol.
+- ✍️ Log in with Audius
+- 🎵 Fetch and stream tracks
+- 🔍 Search and display users, tracks, and playlists
+
+👷‍♀️ We're actively working on building out more SDK features and functionality - stay tuned!
+
+## Installation
+
+- [Node.js](#nodejs)
+- [HTML + JS](#html--js)
+
+### Node.js
+
+#### 1. Install the SDK package using your preferred JS package manager
+
+In your terminal, run:
+
+```bash"
+npm install web3 @audius/sdk
```

Comment:
Should we list web3 second?

```suggestion
npm install @audius/sdk web3
```

Reply:
I'm good with either. I was thinking that since web3 is a "dep" maybe it should come first, but no strong opinion

Reply:
Yeah that makes sense, I'm fine with it as is!